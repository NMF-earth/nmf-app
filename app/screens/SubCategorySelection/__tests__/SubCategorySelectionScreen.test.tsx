import React from "react";
import { create } from "react-test-renderer";

import SubCategorySelectionScreen from "../SubCategorySelectionScreen";

it("SubCategorySelectionScreen renders correctly", () => {
  const tree = create(<SubCategorySelectionScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
