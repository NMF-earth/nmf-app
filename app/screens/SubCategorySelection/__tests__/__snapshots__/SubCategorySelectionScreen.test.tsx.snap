// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubCategorySelectionScreen renders correctly 1`] = `
<RCTScrollView
  style={
    {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "paddingHorizontal": 16,
    }
  }
>
  <View>
    <Text.H2
      style={
        {
          "paddingBottom": 16,
          "paddingTop": 20,
        }
      }
    >
      Select a sub-category
    </Text.H2>
    <Tag
      onPress={[Function]}
      text="Coat"
    />
    <Tag
      onPress={[Function]}
      text="Dress"
    />
    <Tag
      onPress={[Function]}
      text="Jeans"
    />
    <Tag
      onPress={[Function]}
      text="Shirt"
    />
    <Tag
      onPress={[Function]}
      text="Shoes"
    />
    <Tag
      onPress={[Function]}
      text="Sweater"
    />
    <Tag
      onPress={[Function]}
      text="T-Shirt"
    />
  </View>
</RCTScrollView>
`;
